packages/exp/src/commands/build-webhook.js

```diff
@@ -0,0 +1,108 @@
+import { URL } from 'url';
+import { BuildWebhook, Exp } from 'xdl';
+import chalk from 'chalk';
+
+import log from '../log';
+
+export default program => {
```

Comment:
We're about to ship Expo CLI and deprecate exp in the next few days, no new commands should be added to exp anymore. Please define this command in the `expo_commands` folder instead of `commands`. This way it's only included in Expo CLI and not in exp.